src/containers/pages/FocusInvestigation/jurisdiction/index.tsx

```diff
@@ -0,0 +1,370 @@
+  FI_STATUS,
+  FOCUS_AREA_HEADER,
+  NAME,
+  NEXT,
+  PREVIOUS,
+  REACTIVE,
+  ROUTINE_TITLE,
+  START_DATE,
+  STATUS_HEADER,
+} from '../../../../configs/lang';
+import {
+  FIClassifications,
+  locationHierarchy,
+  planStatusDisplay,
+} from '../../../../configs/settings';
+import { CASE_TRIGGERED, FI_SINGLE_MAP_URL, FI_SINGLE_URL } from '../../../../constants';
+import { displayError } from '../../../../helpers/errors';
+import { renderClassificationRow } from '../../../../helpers/indicators';
+import {
+  defaultTableProps,
+  extractPlan,
+  FlexObject,
+  getLocationColumns,
+  transformValues,
+} from '../../../../helpers/utils';
+import supersetFetch from '../../../../services/superset';
+import plansReducer, {
+  fetchPlans,
+  FetchPlansAction,
+  InterventionType,
+  makePlansArraySelector,
+  Plan,
+  reducerName as plansReducerName,
+} from '../../../../store/ducks/plans';
+import reducer, { Message, reducerName, sendMessage } from '../../../../store/tests/ducks/messages';
+import JurisdictionList from './JurisdictionList';
+
+reducerRegistry.register(reducerName, reducer);
+reducerRegistry.register(plansReducerName, plansReducer);
+
+/** Interface for props that come from the URL */
+export interface RouteParams {
+  jurisdictionId: string;
+}
+
+/** Interface for FIJurisdiction component props */
+export interface FIJurisdictionProps {
+  caseTriggeredPlans: Plan[] | null;
+  fetchPlansActionCreator: typeof fetchPlans;
+  loadMessages: typeof sendMessage;
+  messages: Message[];
+  routinePlans: Plan[] | null;
+  supersetService: typeof supersetFetch;
+}
+
+/** default props for ActiveFI component */
+export const defaultActiveFIProps: FIJurisdictionProps = {
+  caseTriggeredPlans: null,
+  fetchPlansActionCreator: fetchPlans,
+  loadMessages: sendMessage,
+  messages: [] as Message[],
+  routinePlans: null,
+  supersetService: supersetFetch,
+};
+
+/** This component is meant to display a list of focus investigations for a
+ * given jurisdiction.  It is meant to display for the focus area i.e. the
+ * lowest level jurisdiction
+ */
+const FIJurisdiction = (props: FIJurisdictionProps & RouteComponentProps<RouteParams>) => {
+  const [loading, setLoading] = useState(true);
+
+  const { caseTriggeredPlans, fetchPlansActionCreator, routinePlans, supersetService } = props;
+
+  const jurisdictionId =
+    '59ad4fa0-1945-4b50-a6e3-a056a7cdceb2' || props.match.params.jurisdictionId;
+
+  // this gets FI plans for the current jurisdiction
+  const supersetParams = superset.getFormData(2000, [
+    { comparator: jurisdictionId, operator: '==', subject: 'jurisdiction_id' },
+    { comparator: InterventionType.FI, operator: '==', subject: 'plan_intervention_type' },
+  ]);
+
+  useEffect(() => {
+    supersetService(SUPERSET_PLANS_SLICE, supersetParams)
+      .then((result: Plan[]) => {
+        if (result) {
+          fetchPlansActionCreator(result);
+        } else {
+          displayError(new Error('An error occurred'));
```

Comment:
@cKellyDesign @p-netm  Noticed that we don't show any errors in cases of HTPP errors (think 4xx/5xx).  This needs to be generalised some more, and I might do that in this issue.  Ideas welcome.

Reply:
For service errors (4xx/5xx upon fetch), we could parameterize the error messages by passing in props/object to override default error messages based on the response code.

```
const customErrorMessages:ErrorMessages = {
  500: 'Whoops, something went way wrong!',
  401: 'Whoops, looks like you need to log in again!',
  400: () => { return 'Whoops, looks like `${jurisdiction_name}` doesn't have any active tasks`; }
}

Reply:
Ooh let's use the translated string here `AN_ERROR_OCCURRED` 🙏